datastore/elastic_search/transfer.py

```diff
@@ -144,6 +144,7 @@ def __init__(self, source, destination):
         self.es_alias = CHATBOT_NER_DATASTORE.get(self.engine).get('es_alias')
         if self.es_alias is None:
             raise AliasForTransferException()
+        self.disable_replicas = CHATBOT_NER_DATASTORE.get(self.engine).get('disable_replicas_while_transferring')
```

Comment:
Set the default value if not found to a default value to False.

Reply:
default value set to False